week3/index.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite + React</title>
+  </head>
```

Comment:
p5: 프로젝트 내용에 맞게 head에 있는 태그를 바꿔 주시는것도 추천드려요~~

```suggestion
 <head>
    <meta charset="UTF-8" />
    <link rel="icon" type="image/svg+xml" href="/Bowoon.svg" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>보운이의 카드돌이</title>
  </head>
```

Reply:
ㅎㅎhtml부분은 생각을 못했네요 알려주셔서 감사해요!
ps. 보운이의 카드돌이..ㅎㅎ로 바꿨어요 참신한 이름 감사해여 😍

Reply:
ㅋㅋㅋㅋㅋ 카드 놀이였는데 오타가... 반영 감사드려유